scripts/demoScripts/create-demo-data.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo 'ORANGE_DEMO_PATIENTS is:';
+echo $ORANGE_DEMO_PATIENTS;
+
+echo 'looping...';
+for i in `seq 0 10`; do
```

Comment:
Low prio: Assumes fixed number of records. Ideally, would operate on the number of records in `ORANGE_DEMO_PATIENTS`. Presuming possible with `jq`.